README.md

```diff
@@ -76,6 +76,10 @@ Kinvey CLI is distributed as an NPM package. After you install NPM, run the foll
 
 ## Options
 
+* `--color <true|false>`
+
+   Enable/disable colors. To disable use `--no-color` or `--color false`.
```

Comment:
I thought the same. It seems that maybe it's a common thing if you have a boolean option to be able to do either `--myBoolOption false` or `--no-myBoolOption`. At least, that is what yargs provides out of the box. For example, they provide out of the box the `--help` option which is again quite common.

Reply:
I don't really mind having both, but I thought it will be more consistent with the other bool options we have: --silent, --verbose, --suppress-version-check. It makes sense to have the color option as "--no-color".